Docs/source/running_cpp/parameters.rst

```diff
@@ -1432,13 +1432,16 @@ s disabled.
     Which species dumped in this diagnostics.
 
 * ``<diag_name>.<species_name>.variables`` (list of `strings` separated by spaces, optional)
-    List of particle quantities to write to output file.
-    By defaults, all quantities are written to file. Choices are
+    List of particle quantities or species-specific field quantities to write to output file.
+    Choices are
 
     * ``w`` for the particle weight,
 
     * ``ux`` ``uy`` ``uz`` for the particle momentum,
```

Comment:
@EZoni I just realized: isn't this list missing: `x`, `y`, `z`?

Reply:
I'm not sure about that. It seems to refer to this namespace: 
https://github.com/ECP-WarpX/WarpX/blob/b1c9e243f1a00f597dfaaa122597c015d71b67ac/Source/Particles/WarpXParticleContainer.H#L62-L73
I think that the input argument that the doc refers to, `<diag_name>.<species_name>.variables`, is read here:
https://github.com/ECP-WarpX/WarpX/blob/b1c9e243f1a00f597dfaaa122597c015d71b67ac/Source/Diagnostics/ParticleDiag/ParticleDiag.cpp#L11-L14
I also thought we could store the particle positions, but this `variables` input seems to be matched with the elements in the namespace `ParticleStringNames` above (which does not include `x`, `y` and `z`):
https://github.com/ECP-WarpX/WarpX/blob/b1c9e243f1a00f597dfaaa122597c015d71b67ac/Source/Diagnostics/ParticleDiag/ParticleDiag.cpp#L33-L48

Reply:
There is an additional comment in the same part of the doc that you pointed at, stating that the particle positions are always included. I guess that is why they were not mentioned among the optional arguments.

Reply:
Yes, I think that's the reason. Particle positions (and id) are part of the AoS data, the variables here describe additional SoA data.